docs/DeveloperGuide.md

```diff
@@ -365,22 +363,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
+* 2a. The list is empty.
+
+    Use case ends.
+
 * 3a. The index given is invalid
-    
+
     * 3a1. bookmark shows an error message
-    
```

Comment:
Just saw that the extension numbering here is incorrect, should be 2a, 2a1, 2b etc. Can you help change them too 🙏 thanks!